cms/static/js/views/list_item.js

```diff
@@ -22,6 +22,7 @@ define([
         canDelete: false,
 
         initialize: function() {
+            this.restrictEditing = this.options.restrictEditing || false;
```

Comment:
The default values are here in case this object is used elsewhere.  I tried to make it so that restrictEditing was not a required option.  This resulted in me not having to change all usages of these classes.